dydx3/modules/public.py

```diff
@@ -266,3 +269,24 @@ def verify_email(
                 'token': token,
             }
         )
+
+    def get_public_retroactive_mining_rewards(
+        self,
+        ethereum_address,
+    ):
+        '''
+        Get public retroactive mining rewards
+
+        :param ethereumAddress: optional
```

Comment:
is this optional? You don't have =None specified in args

Reply:
oops thats required, sorry